src/main/java/Functional.java

```diff
@@ -0,0 +1,29 @@
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class Functional {
+    static int sum(List<Integer> list) {
+        int sum = 0;
+        for (int i : list) {
+            sum += i;
+        }
+        return sum;
+    }
+
+    static int max(List<Integer> list)  throws RuntimeException {
```

Comment:
That is not exactly what I meant.
There are two types of exceptions, they are `Checked` and `Unchecked`.

`Checked` exceptions are exceptions you have to either handle or throw further. For instance, `IOException` is `Checked`, your code won't compile if it uses some code that throws `IOException` and does not handle it throw further.

`Unchecked` exceptions are exceptions that are not necessary to handle at all and define in the method signature. All the subclasses of `RuntimeException` are `Unchecked` you don't need to write them in the method definition or catch, cause it is not always possible.